test/functional/test-custom-element.js

```diff
@@ -1025,7 +1025,7 @@ describes.realWin('CustomElement', {amp: true}, env => {
       expect(element2).to.have.class('i-amphtml-hidden-by-media-query');
     });
 
-    it('should apply sizes condition', () => {
+    it.skip('should apply sizes condition', () => {
```

Comment:
Given the failure mode, I think this can be disabled on Chrome only? Also, add a TODO with the issue and the person who needs to look at this.

```
// TODO(12345, someone): Flaky on Chrome 71.
```

Reply:
This failed on `gulp test --unit`, not in SauceLabs

Reply:
Right, unit tests run on Sauce labs too. (Fewer browsers).